app/api/resources/mentorship_relation.py

```diff
@@ -59,6 +62,20 @@ def is_valid_data(data):
 
         return {}
 
+    @staticmethod
+    def send_email_request(user_id,data):
+        print(data)
+        sender_id = user_id
+        #assume sender is mentor, reciever is mentee
+        sender_role = "mentor"
+        reciever_id = data['mentee_id']
+        #otherwise sender is mentee, reciever is mentor
+        if user_id != data['mentor_id']:
+            sender_role = "mentee"
+            reciever_id = data['mentor_id']
+        send_email_request_notification(
```

Comment:
is this well indented? If yes, then can you add a empty line above this line, for better readability :) if not, please fix the indentation

Reply:
> is this well indented? If yes, then can you add a empty line above this line, for better readability :) if not, please fix the indentation

I added a line above ```send_email_request_notification(```
Was I supposed to add it below?

Reply:
Its great that way @daksha-aeer 🎉